analytics/adobe-analytics/experience-accelerator/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/evolv-exp-acc-aa.js",
   "scripts": {
     "build": "rollup --config ./rollup.config.js",
-    "test": "jest"
+    "test": "jest --passWithNoTests"
```

Comment:
I added this flag to each package that didn't have any tests so that `npm run test` can be run. It was failing on other packages before getting to `/segment/*`